### PR TITLE
GUVNOR-3324: Remove Organizational Unit (Team) input from Advanced Import wizard

### PR DESCRIPTION
**Pull request by mcimbora**
@manstis Removing Org Unit section from the Advanced import wizard, as the wizard is always opened within known "Team". Renamed the first step (Repository) into "Source Repository", and the last step (Org unit) to "Target Repository".

### CODE REVIEW

File: kie-wb-common-screens/kie-wb-common-examples-screen/kie-wb-common-examples-screen-client/src/test/java/org/kie/workbench/common/screens/examples/client/wizard/pages/targetrepository/TargetRepositoryPageTest.java [hunk 1]
```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
```
Comment (manstis) [inline kie-wb-common-screens/kie-wb-common-examples-screen/kie-wb-common-examples-screen-client/src/test/java/org/kie/workbench/common/screens/examples/client/wizard/pages/targetrepository/TargetRepositoryPageTest.java]: 2017

**Comment by mcimbora** [comment]
@manstis Thanks for taking a look. I discussed this with @Rikkola the last week and we agreed the Org unit input can go... I trust your review enough ;-)

**Comment by jhrcek** [comment]
@mcimbora It seems that this removal is breaking selenium tests, which are importing some testing projects via UI. Could you please check and fix them? It should be enough to remove the org-unit-setting logic (and corresponding part of framework)..

**Comment by paulovmr** [comment]
@jhrcek @mcimbora I'm already fixing the tests.

**Comment by manstis** [comment]
@paulovmr I think I merged a PR from @jhrcek fixing this today already.

**Pull request merged**